frontend/components/moduledetails/OccurrenceDetail.jsx

```diff
@@ -0,0 +1,53 @@
+import ImageCard from "../modulecards/ImageCard";
+
+export default function OccurrenceDetail({occurrence}) {
+  return (
+    <>
+      {/*<a href="{% url 'admin:occurrences_occurrence_change' occurrence.pk %}"*/}
+      {/*   target="_blank" className="edit-object-button" rel="noopener noreferrer"*/}
+      {/*   style={{display: "inline-block", position: "absolute", top: "1px", right: "-2rem", fontWeight: "bold"}}>*/}
+      {/*  <i className="fa fa-edit"/>*/}
+      {/*</a>*/}
+
+      {occurrence['verified'] || (
```

Comment:
This is probably nitpicking but is there a better way of doing this for easier reading? It was hard to understand at first that it's checking for the boolean field in `verified`

Reply:
Unfortunately, I think this is just a quirk of JSX syntax. This is the only way to do inline conditional rendering.
https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

The alternative would be do define a variable above:
```jsx
const unverifiedLabel = occurrence["verified"] ? undefined : <span> ... </span>;
```
And then to reference `{unverifiedLabel}` in the return JSX. But then it would not be immediately clear when looking at the return JSX that the label is conditionally rendered.

Reply:
https://modularhistory.atlassian.net/browse/MH-168?focusedCommentId=10240